kroxylicious-operator/src/main/resources/META-INF/fabric8/record-encryptions.filter.kroxylicious.io-v1.yml

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
Possible not a thought for now, but logically, the filter CRD (or at least the OpenAPI def) belongs with the filter implementation rather than the operator.  We could have some code gen that create the filter's config classes and the CRD